packages/slice-machine/components/UpdateModal/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
```

Comment:
I think this component should have a more explicit name like NewVersionModal or NewVersionAvailableModal